core/models/document.py

```diff
@@ -78,6 +80,13 @@ class Meta:
         null=True,
     )
 
+    def get_absolute_url(self) -> str:
+        model = self.content_type.model
+        return reverse(model, args=[str(self.object_id)])
+
+    def display_name(self) -> str:
+        return self.content.name.split("/")[1]
```

Comment:
Would it make sense to just return the filename here? Current implementation returns the first folder in the path to the object.

```suggestion
        return self.shortname
```

Reply:
should we also add the entity name for the display to be like "EPIC-DREM | test.xml"? because with the file name only it feels like a link to the file itself not to the project/contract details page